### PR TITLE
Modified method for consuming redux using hooks

### PR DESCRIPTION
**Pull request by arieltapia16**
- Modified actions and reducer using Constants
- useSelector instead of mapStateToProps
- useDispatch instead of connect with actions
- Added reduxDevTools
Screen Shot 2021-05-22 at 17 17 10

**Use always yarn start instead of npm start**

### CODE REVIEW

File: src/components/ProductList/ProductList.js [hunk 1]
```diff
@@ -37,12 +43,12 @@ const ProductList = ({ selectedFilter, getProducts, productsList }) => {
       ) : (
         <></>
       )}
-      {productsList.loading ? (
+      {list.loading ? (
```
Comment (lucianaDilizio) [inline src/components/ProductList/ProductList.js]: I didn't test it yet, but I think that this should be just `loading`. This is also a note for me, I will check this.

**Comment by lucianaDilizio** [comment]
Thank you very much @arieltapia16! This is so helpful. I will be working on this improvements 💪

File: src/components/ProductList/ProductList.js (hunk 1 above)
Comment (arieltapia16) [reply src/components/ProductList/ProductList.js]: You are right, just "loading"

Comment (arieltapia16) [reply src/components/ProductList/ProductList.js]: Done

**Comment by lucianaDilizio** [comment]
@arieltapia16 I have added two more commits:
- One to apply changes to use hooks with redux in filters 
- The other one, to apply changes to use hooks with redux in cart

**Pull request merged**